### ISSUE TITLE
Ability to pause before starting to write again?

### ISSUE THREAD

**Issue by svennerberg** [mattboldt/typed.js#498]
### Description

I would like to be able to pause right after the last string is deleted and the next string is written. It would be nice to have a property that does this. 

I've tried to create the behavior with the code snippet below using the event preStringTyped. It almost works, only problem is that it still writes out the first character of the new string before pausing.

```
new Typed("#typed", {
  strings: ['Some <i>strings</i> with', 'Another string', 'Chars &times; &copy;'],
  loop: true,
  preStringTyped: (arrayPos, self) => {
    self.stop()

    setTimeout(() => {
      self.start()
    }, 500)
  }
});
```

### Demo
[Demo](https://jsfiddle.net/jrzs0b7a/5/)

**Comment by ComedyTomedy** [comment]
Does this do what you want?

    strings: ['Some <i>strings</i> with', '^500Another string', '^500Chars &times; &copy;'],

If so, you might like to define a function

```
const interposePause = ms => a => a.map((x,i) => i ? `^${ms}${x}` : x)
```

Then use `strings: interposePause(500)(["string1", "string2", "etc"])`

**Comment by sultanarun** [comment]
Add an invisible character at the start : )
https://www.editpad.org/tool/invisible-character

**Comment by cleverhare** [comment]
var typed = new Typed("#typed", {
    stringsElement: '#typed-strings',
    typeSpeed: 0,
    backSpeed: 0,
    backDelay: 500,
    startDelay: 1000,
    loop: false,
    onBegin: function(self) { prettyLog('onBegin ' + self) },
    onComplete: function(self) { prettyLog('onCmplete ' + self) },
    preStringTyped: function(pos, self) { prettyLog('preStringTyped ' + pos + ' ' + self); },
    onStringTyped: function(pos, self) { prettyLog('onStringTyped ' + pos + ' ' + self) },
    onLastStringBackspaced: function(self) { prettyLog('onLastStringBackspaced ' + self) },
    onTypingPaused: function(pos, self) { prettyLog('onTypingPaused ' + pos + ' ' + self) },
    onTypingResumed: function(pos, self) { prettyLog('onTypingResumed ' + pos + ' ' + self) },
    onReset: function(self) { prettyLog('onReset ' + self) },
    onStop: function(pos, self) { prettyLog('onStop ' + pos + ' ' + self) },
    onStart: function(pos, self) { prettyLog('onStart ' + pos + ' ' + self) },
    onDestroy: function(self) { prettyLog('onDestroy ' + self) }
  });
